API/generic/README.md

```diff
@@ -0,0 +1,48 @@
+generic API
+===========
+
+#### `id`
+
+> Returns the identity of the Peer
+
+##### `Go` **WIP**
+
+##### `JavaScript` - ipfs.id([callback])
+
+`callback` must follow `function (err, identity) {}` signature, where `err` is an error if the operation was not successful. `identity` is an object with the Peer identity.
+
+If no `callback` is passed, a promise is returned.
+
+Example:
+
+```js
+ipfs.id(function (err, identity) {
+  if (err) {
+    throw err
+  }
+  console.log(identity)
+})
+```
+
+#### `version`
+
+> Returns the implementation version
+
+##### `Go` **WIP**
+
+##### `JavaScript` - ipfs.version([callback])
+
+`callback` must follow `function (err, version) {}` signature, where `err` is an error if the operation was not successful. `version` is an object with the version of the implementation, the commit and the Repo.
+
+If no `callback` is passed, a promise is returned.
+
+Example:
+
+```js
+ipfs.version(function (err, version) {
```

Comment:
They never really worked on the http-api level https://github.com/ipfs/js-ipfs-api/blob/master/test/interface-ipfs-core/version.spec.js#L8-L9, the filtering just happens on the CLI.